apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -170,10 +169,9 @@ class SectionProgress extends Component {
 export const UnconnectedSectionProgress = SectionProgress;
 
 export default connect(state => ({
-  scriptId: state.sectionProgress.scriptId,
+  scriptId: state.scriptSelection.scriptId,
   section: state.sectionProgress.section,
-  validScripts: state.sectionProgress.validScripts,
-  studentScriptIds: state.sectionProgress.studentScriptIds,
+  validScripts: state.scriptSelection.validScripts,
```

Comment:
Given that we now have a reducer that is specific to the script selector, it would be ideal for the ScriptSelector to become a connected component and supply the defaultScriptId and validScripts props to it directly. It could be fine to do this in a later PR.